app/services/invoices/create_service.rb

```diff
@@ -129,13 +130,21 @@ def applied_coupon
       @applied_coupon = customer.applied_coupons.active.first
     end
 
+    def credit_notes
+      customer.credit_notes.available
```

Comment:
What do you think about memoizing credit notes and sending it to `Credits::CreditNoteService`?